packages/cfd/src/Constants/cfd_compare_account_content.ts

```diff
@@ -0,0 +1,226 @@
+            financial_bvi: {
+                text: localize('British Virgin Islands Financial Services Commission (licence no. SIBA/L/18/1114)'),
+            },
+            financial_vanuatu: {
+                text: localize('Vanuatu Financial Services Commission'),
+            },
+            financial_labuan: { text: localize('Labuan Financial Services Authority (Licence no. MB/18/0024)') },
+            derivx: { text: 'Financial Commission' },
+        },
+    },
+    {
+        id: 'leverage',
+        attribute: localize('Maximum leverage'),
+        values: {
+            synthetic_svg: { text: localize('Up to 1:1000') },
+            synthetic_bvi: { text: localize('Up to 1:1000') },
+            financial_vanuatu: { text: localize('Up to 1:1000') },
+            financial_labuan: { text: localize('Up to 1:100') },
+            derivx: { text: localize('Up to 1:1000') },
+        },
+    },
+    {
+        id: 'instruments',
+        attribute: localize('Trading instruments'),
+        values: {
+            synthetic_svg: { text: [localize('Synthetics'), localize('Baskets'), localize('Derived FX')] },
+            financial_svg: {
+                text: [
+                    localize('Forex: standard/micro'),
+                    localize('Stocks'),
+                    localize('Stock indices'),
+                    localize('Commodities'),
+                    localize('Cryptocurrencies'),
+                ],
+            },
+            financial_vanuatu: {
+                text: [
+                    localize('Forex'),
+                    localize('Stock indices'),
+                    localize('Commodities'),
+                    localize('Cryptocurrencies'),
+                ],
+            },
+            financial_labuan: { text: [localize('Forex'), localize('Cryptocurrencies')] },
+            derivx: {
+                text: [
+                    localize('Synthetics'),
+                    localize('Baskets'),
+                    localize('Forex: standard/micro'),
+                    localize('Stocks'),
+                    localize('Stock indices'),
+                    localize('Commodities'),
+                    localize('Cryptocurrencies'),
+                ],
+            },
+        },
+    },
+];
+
+export const cr_real_footer_buttons: TCompareAccountFooterButtonData[] = [
+    { label: localize('Add'), action: 'synthetic_svg' },
+    { label: localize('Add'), action: 'synthetic_bvi' },
+    { label: localize('Add'), action: 'financial_svg' },
+    { label: localize('Add'), action: 'financial_bvi' },
+    { label: localize('Add'), action: 'financial_vanuatu' },
+    { label: localize('Add'), action: 'financial_labuan' },
+    { label: localize('Add'), action: 'derivx' },
+];
+export const eu__real_footer_button: TCompareAccountFooterButtonData[] = [
+    { label: localize('Add'), action: 'financial_maltainvest' },
+];
+
+export const preappstore_demo_cr_content: TCompareAccountContentProps[] = [
```

Comment:
to be consistent with other variables I suggest to change it to
```suggestion
export const preappstore_cr_demo_content: TCompareAccountContentProps[] = [
```